BaseApi/Transformer/AbstractTransformer.php

```
@@ -82,4 +92,20 @@ public function transform($mixed)
     public function reverseTransform(FacadeInterface $facade, $source = null)
     {
     }
+
+    /**
+     * @throws TransformerParameterTypeException
+     *
+     * @return FacadeInterface
+     */
+    protected function newFacade()
+    {
+        $facade = new $this->facadeClass();
```

Maybe you should throw an exception if the property `facadeClass` is not set ?